data_manipulation.qmd

```diff
@@ -368,20 +367,20 @@ pums_person <- open_dataset("./data/person")
 
 ```{r}
 #| label: fix-lookup-join
-pums_person |>
-  mutate(year = cast(year, int16())) |>
-  left_join(
-    mean_commute_time_arrow,
-    by = c("location", "year")
-  )|>
-  mutate(commute_diff = mean_commute_time - JWMNP) |>
-  select(
-    commute_diff,
-    commute_time = JWMNP,
-    mean_commute_time
-  ) |>
-  head(3) |>
-  collect()
+ pums_person |>
```

Comment:
Is this right? Looks like an extra space was added.

Reply:
Fixed